exportSkybox.py

```diff
@@ -32,7 +32,7 @@
 
 bl_info = {
     "name": "Export Skybox",
-    "description": "Export a Skybox",
+    "description": "Add-on to easily export skybox-ready images of a scene.",
```

Comment:
This line is shown in Blender UI, can you please remove "Add-on to" from the string and then it's OK to be merged.

Reply:
Done!